examples/700hPa_RELH_Winds.py

```diff
@@ -0,0 +1,127 @@
+"""
+700-hPa Relative Humidity
+=========================
+
+Classic 700-hPa Map that display Relative Humidity
+
+By: Kevin Goebbert
+
+This example uses GFS output to display common 700-hPa parameters
+including relative humidity.
+
+"""
+
+
+######################################################################
+# Import needed modules
+#
+
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+import metpy.calc as mpcalc
+from metpy.units import units
+import numpy as np
+import xarray as xr
+
+
+######################################################################
+# Access data using Xarray
+#
+
+ds = xr.open_dataset('https://thredds.ucar.edu/thredds/dodsC/casestudies/'
+                     'python-gallery/GFS_20101026_1200.nc')
+
+
+######################################################################
+# Data Parsing
+# ------------
+#
+# Parse out desired data and attach units.
+#
+# Smooth using MetPy’s ``smooth_gaussian`` function to minimize noise in
+# output.
+#
+
+# Set subset slice for the geographic extent of data to limit download
+lon_slice = slice(200, 350)
+lat_slice = slice(85, 10)
+
+# Grab lat/lon values (GFS will be 1D)
+lats = ds.lat.sel(lat=lat_slice).values
+lons = ds.lon.sel(lon=lon_slice).values
+
+# Select specific level data
+hght_700 = mpcalc.smooth_gaussian(ds['Geopotential_height_isobaric'].sel(
```

Comment:
Can you test with a straight ‘DataArray’? Then MetPy might be able to handle units for you.